sqlmesh/core/engine_adapter/snowflake.py

```diff
@@ -422,3 +433,49 @@ def _create_column_comments(
                 f"Column comments for table '{table.alias_or_name}' not registered - this may be due to limited permissions.",
                 exc_info=True,
             )
+
+    def get_alter_expressions(
```

Comment:
Is this something we can only do for snowflake?

Reply:
For now, I didn't investigate the possibility or syntax on other databases

Reply:
Can we please follow-up with the same support for BigQuery

Reply:
https://github.com/TobikoData/sqlmesh/pull/3231